docs/distribution.md

```diff
@@ -0,0 +1,19 @@
+# Project
+
+## Contents
+-[blah](architecture.md)
```

Comment:
Hmm, this came in through someone else's push. I will fix them but I prefer to do that when I do my tests with the build fixes.